client/src/views/home/DeviceJoinOrganizationModal.vue

```diff
@@ -329,24 +349,54 @@ async function nextStep(): Promise<void> {
       waitingForHost.value = false;
       pageStep.value += 1;
     } else {
-      console.log('Wait peer trust failed', result.error);
+      await showErrorAndRestart(t('ClaimDeviceModal.errors.unexpected', {reason: result.error.tag}));
     }
   }
 }
 
-onMounted(async () => {
+async function startProcess(): Promise<void> {
+  pageStep.value = DeviceJoinOrganizationStep.Information;
+  deviceName.value = '';
+  waitingForHost.value = true;
+
   const retrieveResult = await claimer.value.retrieveInfo(props.invitationLink);
 
-  if (retrieveResult.ok) {
-    const waitResult = await claimer.value.initialWaitHost();
-    if (waitResult.ok) {
-      waitingForHost.value = false;
-    } else {
-      console.log('Wait host failed', waitResult.error);
-    }
-  } else {
-    console.log('Failed to retrieve info', retrieveResult.error);
+  if (!retrieveResult.ok) {
+    await notificationCenter.showModal(new Notification({
+      message: t('ClaimDeviceModal.errors.startFailed'),
+      level: NotificationLevel.Error,
+    }));
+    await cancelModal();
+    return;
+  }
+
+  const waitResult = await claimer.value.initialWaitHost();
+  if (!waitResult.ok) {
+    await notificationCenter.showModal(new Notification({
+      message: t('ClaimDeviceModal.errors.startFailed'),
+      level: NotificationLevel.Error,
+    }));
+    await cancelModal();
+    return;
+  }
```

Comment:
The same code is repeated 2 times in a row, should we use a logic operator `OR` with the two `const` above?

`const retrieveResult = await claimer.value.retrieveInfo(props.invitationLink);`
`const waitResult = await claimer.value.initialWaitHost();`


`if (!retrieveResult.ok || !waitResult.ok) {
...
}`